apps/test/util/testUtils.js

```diff
@@ -161,6 +161,10 @@ exports.debugMode = function () {
  * @param {number} top Vertical offset from top left of visualization to drop at
  */
 exports.dragToVisualization = function (type, left, top) {
+  // this delta of 40 is a bit of a mystery, but I think it has to do with our
+  // tests not having CSS applied to them.
+  left = left + 40;
```

Comment:
I'm a little skeptical that CSS would not be being applied, partially since it seems like the most likely explanation for everything being at 0.5 scale. Fine to leave this comment as-is since we don't really know, though.